guides/source/index.html.erb

```diff
@@ -14,11 +14,24 @@
     <% end -%>
     <dd class="work-in-progress">Guides marked with this icon are currently being worked on and will not be available in the Guides Index menu. While still useful, they may contain incomplete information and even errors. You can help by reviewing them and posting your comments and corrections.</dd>
   </dl>
+  <h3 class="chapter"><img src="images/chapters_icon.gif" alt="" />Guides</h3>
```

Comment:
```suggestion
  <h3 class="chapter"><img src="images/chapters_icon.gif"/>Guides</h3>
```

Reply:
Hey @marivaldo 👋  Thanks for the suggestion!

I copied this from the other guides, which use it for the `Chapters` icon. 
Looking at the W3C requirements, they recommend using an empty alt for decorative images.
It prevents screen readers from announcing the file name of the image:

> In these cases, a null (empty) alt text should be provided (alt="") so that they can be ignored by assistive technologies, such as screen readers. Text values for these types of images would add audible clutter to screen reader output or could distract users if the topic is different from that in adjacent text. Leaving out the alt attribute is also not an option because when it is not provided, some screen readers will announce the file name of the image instead.
https://www.w3.org/WAI/tutorials/images/decorative/

Reply:
> Hey @marivaldo 👋 Thanks for the suggestion!
> 
> I copied this from the other guides, which use it for the `Chapters` icon.
> Looking at the W3C requirements, they recommend using an empty alt for decorative images.
> It prevents screen readers from announcing the file name of the image:
> 
> > In these cases, a null (empty) alt text should be provided (alt="") so that they can be ignored by assistive technologies, such as screen readers. Text values for these types of images would add audible clutter to screen reader output or could distract users if the topic is different from that in adjacent text. Leaving out the alt attribute is also not an option because when it is not provided, some screen readers will announce the file name of the image instead.
> > https://www.w3.org/WAI/tutorials/images/decorative/

Hi @p8, cool, I didn't know about the recommendation, learning more from you. 😄 
That's ok